packages/@okta/vuepress-site/docs/guides/oin-universal-logout-overview/index.md

```diff
@@ -36,11 +36,47 @@ The actual endpoint URL is up to the discretion of the app developer building th
 
 ### Endpoint authentication
 
-The request to the Universal Logout endpoint requires authentication so that your app knows the request is coming from Okta. Okta supports the [JWT with private key](https://developer.okta.com/docs/api/openapi/okta-oauth/guides/client-auth/#jwt-with-private-key)(`private_key_jwt`) client authentication method.
+The request to the Universal Logout endpoint requires authentication so that your app knows the request is coming from Okta. Okta sends a signed JWT to authenticate to your API. The JWT follows a similar format to the [`private_key_jwt`](https://developer.okta.com/docs/api/openapi/okta-oauth/guides/client-auth/#jwt-with-private-key) format used as OAuth 2.0 client authentication. The details of the JWT claims are described below. The format `<>` indicates a placeholder value and isn't included as part of a value.
+
+The JWT is sent using the `Bearer` HTTP Authorization scheme:
+
+```BASH
+Authorization: Bearer <JWT>
+```
+
+The claims of the JWT are the following:
+
+```JSON
+// Header
+{
+  "typ": "global-token-revocation+jwt",
+  "alg": "<algorithm used for signing SSO token>"
+}
+// Payload
+{
+  "jti": "<unique identifier>",
+  "iss": "<orgDomainBaseUrl/customDomainBaseUrl>",
+  "sub": "<client_id of OIDC app/appInstanceId of SAML 2.0 app>",
+  "aud": "<revocation endpoint URL>",
+  "exp": "<5 min into future>",
+  "nbf": "<5 min ago>",
+  "iat": "<current timestamp>"
+}
+```
+
+* `jti`: A unique identifier for this JWT
+* `iss`: The same issuer URL that you receive in an OpenID Connect ID token
+* `sub`: Identifies the "subject" of this token, which in this case is your application. For OpenID Connect clients this is the `client_id`, and for SAML integrations, it's the `appInstanceId`.
+* `aud`: Identifies the "audience" of this token, which is the URL of your Global Token Revocation endpoint. The URL doesn't include query string parameters or a URL fragment.
+* `exp`: The expiration timestamp of the token, which is 5 minutes.
+* `nbf`: A timestamp of 5 minutes in the past.
+* `iat`: The current timestamp of when the token was created.
+
+Your API endpoint should validate the signature of the JWT and these claims to confirm the revocation request is coming from Okta. The token is signed with the same key used for signing ID tokens or SAML assertions for SSO.
 
 ### Logout request
 
-When a user should be logged out of the app, Okta makes a POST request to the Universal Logout endpoint. The request includes a JSON object in the request body that describes the user to be logged out.
+When a user should be signed out of the app, Okta makes a POST request to the Universal Logout endpoint. The request includes a JSON object in the request body that describes the user to be logged out.
```

Comment:
```suggestion
When a user should be signed out of the app, Okta makes a POST request to the Universal Logout endpoint. The request includes a JSON object in the request body that describes the user.
```

Reply:
You can ignore this if you want; just thinking that it could be cut because I can't think of any other user description it would contain but the one you just mentioned. If you do keep it, recommend changing the final instance of "logged out" to "signed out" for consistency.